add-on/panels/css/panel.css

```diff
@@ -96,6 +96,56 @@ body {
   border-radius: 3px;
 }
 
+/* Rename closing room panel */
+.rename-container {
+  margin: 15px;
+  padding-left: 45px;
+}
+
+.rename-newRoom img {
+  margin-left: 15px;
+  position: absolute;
+  width: 30px;
+}
+
+.rename-header {
+  color: #000;
+  font-size: 1.2em;
+  padding: 5px 0;
+}
+
+.rename-prompt {
+  margin-bottom: 5px;
+}
+
+.input-group {
```

Comment:
This needs scoping to `.rename-container > .input-group` or `.rename-newRoom > .rename-container > .input-group`.

At the moment, it runs the risk of affecting the share panel view.